core/src/main/java/com/linecorp/armeria/common/HttpHeaderNames.java

```diff
@@ -70,6 +70,7 @@
     private static final BitSet PROHIBITED_NAME_CHARS;
     private static final String[] PROHIBITED_NAME_CHAR_NAMES;
     private static final byte LAST_PROHIBITED_NAME_CHAR;
+    private static final ImmutableMap.Builder<AsciiString, String> inverseMapBuilder = ImmutableMap.builder();
```

Comment:
Could we set `null` to this field after calling `.build()`, so we're sure nobody calls it after that point?